ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -93,12 +94,7 @@ object LedgerApiTestTool {
     }
 
     if (config.extract) {
```

Comment:
We may be able to get rid of this option entirely, or at least deprecate it, if uploading the DARs is the only use case. However, it's probably useful for debugging too (at least), so I didn't want to do that without asking.